rt-forecast/format-rt-fn.R

```diff
@@ -74,28 +76,38 @@ format_rt_forecast <- function(loc = NULL, loc_name = NULL,
 
 # Get estimates ---------------------------------------------------------------
 
+  file_loc <- paste0(loc, "/", forecast_date)
   
```

Comment:
Fixed

Reply:
interesting - the previous version didn't work on your computer?